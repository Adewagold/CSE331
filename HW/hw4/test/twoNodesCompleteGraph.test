# Test adding two nodes, two reflexive edges 
# and two edges between two nodes.

# Create a graph
CreateGraph two_nodes_complete_graph

# Add two nodes
AddNode two_nodes_complete_graph a
AddNode two_nodes_complete_graph b

# Add four edges (including tow reflexive edges)
AddEdge two_nodes_complete_graph a a AA
AddEdge two_nodes_complete_graph a b AB
AddEdge two_nodes_complete_graph b a BA
AddEdge two_nodes_complete_graph b b BB

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_complete_graph
ListChildren two_nodes_complete_graph a
ListChildren two_nodes_complete_graph b