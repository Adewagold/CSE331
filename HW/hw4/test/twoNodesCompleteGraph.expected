# Test adding two nodes, two reflexive edges 
# and two edges between two nodes.

# Create a graph
created graph two_nodes_complete_graph

# Add two nodes
added node a to two_nodes_complete_graph
added node b to two_nodes_complete_graph

# Add four edges (including tow reflexive edges)
added edge AA from a to a in two_nodes_complete_graph
added edge AB from a to b in two_nodes_complete_graph
added edge BA from b to a in two_nodes_complete_graph
added edge BB from b to b in two_nodes_complete_graph

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_complete_graph contains: a b
the children of a in two_nodes_complete_graph are: a(AA) b(AB)
the children of b in two_nodes_complete_graph are: a(BA) b(BB)